autoload/airline/extensions/coc.vim

```diff
@@ -42,7 +42,12 @@ function! airline#extensions#coc#get(type)
   endif
 endfunction
 
+function! airline#extensions#coc#get_status()
```

Comment:
I'm not sure I understand why I'm getting the
> Use the abort attribute for functions in autoload (see Google VimScript Style Guide (Functions))
here when the other functions in this file don't on ci. However, when I run vint locally I get the error for all the functions in this file?

Reply:
Can you just add as a second commit the `abort` attribute to all the other functions? Thanks

Reply:
Sure.